addons/gsri_opex/Stringtable.xml

```diff
@@ -238,28 +242,32 @@
 
 	<Package name="FREMM_submarine">
 		<Key ID="STR_GSRI_FREMM_submarineOk">
-			<Original>Submarine has moved to the designated location (%1).</Original>
-			<French>Le sous-marin s'est déplacé aux coordonnées indiquées (%1).</French>
+			<Original>Submarine has moved to the designated location.</Original>
+			<French>Le sous-marin s'est déplacé aux coordonnées indiquées.</French>
 		</Key>
 		<Key ID="STR_GSRI_FREMM_submarineFail">
-			<Original>Submarine cannot go here.</Original>
-			<French>Le sous-marin ne peut pas se déplacer ici.</French>
+			<Original>Submarine cannot go here, 30m+ depth is required.</Original>
+			<French>Le sous-marin ne peut pas se déplacer ici, profondeur de 30m+ requise.</French>
```

Comment:
```suggestion
			<French>Le sous-marin ne peut pas se déplacer ici, profondeur de 30m requise.</French>
```

Reply:
This + symbol is willingful, its meaning is 30m depth or more. Is it unclear/incorrect ?

Reply:
sea above
/me giggles